index.html

```diff
@@ -19,9 +44,9 @@ <h2 class="title-text">All Awesome Books</h2>
         <tbody>
```

Comment:
- [ ] Excellent job implementing the changes requested by the previous reviewer :clap: . However, the layout of the table body content (the title and author of the book, as well as the remove btn) does not correspond to the wireframe. The title and author of the book are aligned completely to the left in the wireframe, as seen in the screenshot added by the previous reviewer, and the remove button is completely to the right. :+1: 

#### You design :point_down: 
![Screenshot from 2023-05-03 16-39-25](https://user-images.githubusercontent.com/89228743/235983086-49599dec-7958-4755-a316-a268b6f753c2.png)

#### The wireframe :point_down: 
![Screenshot from 2023-05-03 16-40-34](https://user-images.githubusercontent.com/89228743/235983209-38f88980-aed7-4ae6-b680-c39053985334.png)



So one way to fix that is to go into your app.js file and update the `displayBooks` method by adding this :point_down: 
`removeCell.classList.add('align-rm-btn');`
But first, you need to the `style.css` file and add the styling for the `align-rm-btn` class like this :point_down: 
```
.align-rm-btn {
  display: flex;
  justify-content: flex-end;
}
```
After the update the `displayBooks` method in the `app.js` file is going to look like this :point_down: 

```
 displayBooks() {
    this.tableBody.innerHTML = '';
    this.books.forEach((book) => {
      const row = this.tableBody.insertRow();
      const titleCell = row.insertCell();
      const removeCell = row.insertCell();
  👉️    removeCell.classList.add('align-rm-btn');
      const bookTitle = `${book.title} by ${book.author}`;
      titleCell.innerText = bookTitle;
      removeCell.innerHTML = `<button onclick="bookManager.removeBook('${book.id}')">Remove</button>`;
    });
  },
```

Notice the left pointing finger it's the only line we added in the method. If you want to copy that and paste it into your code please make sure to remove the left pointing finger emoji. :ok_hand: 
You can inspect the browser to see what happened here. 🤓

If you go through the suggestion, your result is going to look like this :point_down: 

![Screenshot from 2023-05-03 16-58-39](https://user-images.githubusercontent.com/89228743/235987346-d1a801dd-295e-40c6-8c9a-f6db8760fa22.png)


#### I hope this helps 😊